components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/ChallengeQuestionProcessor.java

```diff
@@ -182,10 +185,12 @@ public void setChallengeQuestions(ChallengeQuestionDTO[] questionDTOs) throws Id
 
         } catch (Exception e) {
             String msg = "No associated challenge question found for the user";
```

Comment:
Two question
1) Any reason for logging an exception only in debug mode. ?
2) Unnecessarily creating a variable even If debug mode is disabled. Please fix.